pages/common/jj-simplify-parents.md

```diff
@@ -0,0 +1,12 @@
+# jj simplify-parents
+
+> Simplify parent edges for the specified revision(s).
```

Comment:
It's not immediately clear to the layman what "simplifying parents" means, we may want to add a short explanation of what that is. (E.g. 'Removes redundant parents from the visible tree (e.g. turns "A -> B -> C | A -> C" into just "A -> B -> C"' or something to that effect).